### ISSUE TITLE
`Tooltip` without text is missing discernable label on icon

### ISSUE THREAD

**Issue by brunnerh** [carbon-design-system/carbon-components-svelte#1094]
The component has an `iconDescription` property, but it is only applied if there also is a trigger text.

This seems the wrong way around for `aria-label` and `aria-labeledby` should possibly always be set.

```js
    "aria-labelledby": triggerText ? triggerId : undefined,
    "aria-label": triggerText ? iconDescription : undefined,
```

If `triggerText` is set, the label would be derived from the content, if not, from the `aria-label` via `iconDescription`.

[REPL example](https://svelte.dev/repl/331cf9baf0a24382884fff4bacf36d34?version=3.46.4)

{Found via Lighthouse)

**Comment by metonym** [comment]
Nice catch – `aria-label` should use `iconDescription` if `triggerText `is falsy.

Switching it gives a 100 Lighthouse score.

```diff
- "aria-label": triggerText ? iconDescription : undefined,
+ "aria-label": triggerText ? undefined : iconDescription,
```

**Comment by metonym** [comment]
Fixed in [v0.58.2](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.58.2)
